site/.knip.jsonc

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://unpkg.com/knip@5/schema.json",
+	"entry": ["./src/index.tsx"],
+	"project": ["./src/**/*.ts", "./src/**/*.tsx"],
+	"ignore": ["**/*Generated.ts", "./src/serviceWorker.ts"],
+	"ignoreBinaries": ["protoc"],
+	"ignoreDependencies": [
+		"express",
+		"@types/express",
+		"ssh2",
+		"@types/ssh2",
+		"@types/react-virtualized-auto-sizer",
+		"jest_workaround",
+		"protobufjs",
+		"rxjs",
+		"ts-proto"
```

Comment:
most of these are things that we use in the e2e tests. we should probably add the e2e director to the `project`.